Detections/SigninLogs/AnomalousIPUsageFollowedByTeamsAction.kql

```diff
@@ -0,0 +1,108 @@
+id: 2b701288-b428-4fb8-805e-e4372c574786
```

Comment:
Since this is across multiple data sources, I think it should live in the MultipleDataSources folder.

Reply:
File should now be in the appropriate directory.